dao/src/main/java/greencity/repository/UserRepo.java

```diff
@@ -220,9 +223,19 @@
     /**
      * Find and return all registration months.
      *
-     * @return {@link List} of months
+     * @return {@link List} of {@link RegistrationStatisticsDtoResponse}
```

Comment:
This JavaDoc would be much useful if it can explain that the query behind this method is the one in 
```
@NamedNativeQuery(name = "User.findAllRegistrationMonths"
```
and how this method identifies it.

Reply:
Done. Does it look good now?

Reply:
Much better now.